mmv1/third_party/terraform/website/docs/r/bigquery_dataset_iam.html.markdown

```diff
@@ -107,6 +179,23 @@ The following arguments are supported:
 * `project` - (Optional) The ID of the project in which the resource belongs.
     If it is not provided, the provider project is used.
 
+* `condition` - (Optional) An [IAM Condition](https://cloud.google.com/iam/docs/conditions-overview) for a given binding.
+  Structure is documented below.
+
+---
+
+The `condition` block supports:
```

Comment:
`google_bigquery_dataset.access` and `google_bigquery_dataset_access` support `condition.location` today: https://github.com/GoogleCloudPlatform/magic-modules/pull/12475. Can we also include it here for parity?